.teamcity/ETL/buildTypes/DeployETL.kt

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+package etl.buildTypes
+
+import jetbrains.buildServer.configs.kotlin.v2019_2.*
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.swabra
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.nuGetPublish
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.powerShell
+
+
+object DeployETLBuild : BuildType ({
+    name = "Deploy"
+
+    features {
+        // Default setting is to clean before next build
```

Comment:
FYI, recently I realized that this default setting is a "swabra clean", not a "git clean". There is subtle difference between the two. The default setting is generally good enough, but occasionally will need to be modified to force a `git clean`. 

No action to take here.